samples/Erlang/release.es

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env escript
```

Comment:
Hm. makes sense :-)
I think it would be preferable to add a new EScript samples rather than editing one though.